src/postfixCompletionProvider.ts

```diff
@@ -49,7 +49,7 @@ export class PostfixCompletionProvider implements vsc.CompletionItemProvider {
     try {
       return this.templates
         .filter(t => {
-          let canUseTemplate = t.canUse(currentNode)
+          let canUseTemplate = t.canUse(ts.isNonNullExpression(currentNode) ? currentNode.expression : currentNode)

```

Comment:
I didn't find more elegant solution (without introducing a lot of code changes). since most checks like `inReturnStatement` traverses recursively it doesn't seem to break anything

Reply:
All tests are passing so I think it doesn't ;)
Not a perfect solution but not very bad as well. Good enough if it fixes the problem. It will be still very easy to improve that later in the future so this looks good to me.